src/Games/NexusMods.Games.AdvancedInstaller.UI/Window/AdvancedInstallerWindowViewModel.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Reactive.Disposables;
+using NexusMods.DataModel.Games;
+using NexusMods.DataModel.ModInstallers;
+using NexusMods.Paths;
+using NexusMods.Paths.FileTree;
+using ReactiveUI;
+using ReactiveUI.Fody.Helpers;
+
+
+namespace NexusMods.Games.AdvancedInstaller.UI;
+
+public class AdvancedInstallerWindowViewModel : AViewModel<IAdvancedInstallerWindowViewModel>,
+    IAdvancedInstallerWindowViewModel
+{
+
+    /// <InheritDoc/>
+    [Reactive] public IViewModelInterface CurrentPageVM { get; protected set; }
+
+    /// <InheritDoc/>
+    public IUnsupportedModPageViewModel UnsupportedModVM { get; }
+
+    /// <InheritDoc/>
+    public IAdvancedInstallerPageViewModel AdvancedInstallerVM { get; }
+
+    /// <summary>
+    /// Construct the VM for the Advanced Installer container window.
+    /// </summary>
+    /// <param name="modName">The name of the mod to install.</param>
+    /// <param name="archiveFiles">The tree of files contained in the archive.</param>
+    /// <param name="register">The register of the game locations.</param>
+    /// <param name="gameName">The name of the game managed.</param>
+    public AdvancedInstallerWindowViewModel(string modName,
+        FileTreeNode<RelativePath, ModSourceFileEntry> archiveFiles,
+        GameLocationsRegister register,
+        string gameName)
+    {
+        AdvancedInstallerVM = new AdvancedInstallerPageViewModel(modName, archiveFiles, register, gameName);
+        UnsupportedModVM = new UnsupportedModPageViewModel(modName);
+        CurrentPageVM = UnsupportedModVM;
+
+        this.WhenActivated(disposables =>
+        {
+            UnsupportedModVM.AcceptCommand
+                .Subscribe(_ => CurrentPageVM = AdvancedInstallerVM)
+                .DisposeWith(disposables);
+        });
```

Comment:
Use `BindToVM` to expose the latest value from an observable stream ([guidelines](https://github.com/Nexus-Mods/NexusMods.App/blob/main/docs/UICodingConventions.md#view-model-properties)).

Reply:
So something like this?
```csharp
            UnsupportedModVM.AcceptCommand
                .Select(_ => AdvancedInstallerVM)
                .BindToVM(this, vm => vm.CurrentPageVM)
                .DisposeWith(disposables);
```

Reply:
> So something like this?

Yes.

Reply:
I think the subscribe is clearer in this case. The stream doesn't represent the current value of the property, so a binding doesn't make much sense to me here. 
Is there a specific reason BindTo would be better than changing the value inside Subscribe()?